### DISCUSSION TITLE
Ipv6 access

### DISCUSSION THREAD

**Discussion by Havard20009** [keivanipchihagh/x-ui#6] [Q&A]
hello
i have a problem with my access from ipv6.
i did everything as you suggested
but i can not access 
network:
    version: 2
    ethernets:
        eth0:
            addresses:
            - **.**.70.**/24
            - 10.48.0.9/16
            - **:1838:37:**::**/64
            match:
                macaddress: 00:**:**:35:b3:**
            mtu: 1500
            nameservers:
                addresses:
                - 8.8.8.8
                - 1.1.1.1
                - 2001:4860:4860::8888
                - 2001:4860:4860::8844
                search: []
            routes:
            -   to: 0.0.0.0/0
                via: **.**.***.*
            -   to: **:1838:**::1
                scope: link
            set-name: eth0

**Comment by keivanipchihagh** [comment]
Hi there,
Can you confirm you have IPv6 on your network setup and functional?

**Reply by Havard20009** [reply]
yes i have already
and i can ping my server


Canada, BC, Vancouver | Telus | 0% | 2 | 189 | 189.2 | 189 | 189.39 | 0.19 | ... |  
USA, CA, Fremont | Linode | 0% | 2 | 178.48 | 178.24 | 178 | 178.48 | 0.24 | ... |  
USA, CA, San Francisco | Digital Ocean | 0% | 2 | 181.79 | 183.69 | 181.79 | 185.59

**Reply by keivanipchihagh** [reply]
In that case, your IPv6 is up and running, and your configurations are correct. What is the problem then?
